CoreLocationDemo/MapViewController.swift

```diff
@@ -0,0 +1,39 @@
+//
+//  MapViewController.swift
+//  CoreLocationDemo
+//
+//  Created by Frakcool on 12/20/18.
+//  Copyright © 2018 Jeremiah Hawks. All rights reserved.
+//
+
+import UIKit
+import MapKit
+import CoreLocation
+
+class MapViewController: UIViewController {
+    @IBOutlet weak var mapView: MKMapView!
+    
+    var placemark: CLPlacemark?
+    let regionRadius: CLLocationDistance = 1000
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        let location = placemark?.location
```

Comment:
I would prefer to see these in a seperate setup type function. It makes the code clearer, and seperates the responsabilities of individual functions. 

In addition, I'd like to see some sort of safe unwrapping of optionals. This is less for you, and more for others who will be using or maintaining your code. Other team members may not take the time to understand your code and maybe hasty when writing their own code. YOU know it's safe, but they may misuse it or not understand how to use it safely.